workflows/files/ZWECONF.xml

```diff
@@ -469,6 +469,17 @@ How we want to verify SSL certificates of services. Valid values are:
             <default>7554</default>
         </integer>
     </variable>
+    <variable name="components_gateway_internalMapper" scope="instance" visibility="public">
```

Comment:
This is not wrong, however I was confused by the name `components_gateway_internalMapper` expecting to find matching property in gateway schema. But there is `component.gateway.apiml.security.useInternalMapper`. The setting then is done at the right place, but consider using corresponding names.

Reply:
Technically it is ID, I kept "same shorten approach" as other variables from same section, like components_gateway_port, components_gateway_debug, but I could update it to "long name". Only issue with long names is, that it is hard to read direct xml, or do quick update, when variables are used in conditions, especially on greenscreen. This one is just used and injected within shell script, so I could sync it with components_gateway_apiml_security_auth_zosmf_jwtAutoconfiguration. But in case when variable is used in some condition, especially when condition is more complex or contain multiple variables, code is almost unreadable with long names, and in inline-jcls have to be shortened anyway.